AirCasting/CreateSessionViews/CreateSessionDetails/CreateSessionDetailsViewModel.swift

```diff
@@ -0,0 +1,81 @@
+// Created by Lunar on 10/11/2021.
+//
+import Foundation
+import SystemConfiguration.CaptiveNetwork
+
+class CreateSessionDetailsViewModel: ObservableObject {
+    
+    @Published var sessionName: String = ""
+    @Published var sessionTags: String = ""
+    @Published var isIndoor = true
+    @Published var isWiFi = true
+    @Published var wifiPassword: String = ""
+    @Published var wifiSSID: String = ""
+    @Published var isConfirmCreatingSessionActive: Bool = false
+    @Published var isLocationSessionDetailsActive: Bool = false
+    @Published var showingAlert = false
+    @Published var isSSIDTextfieldDisplayed: Bool = false
+    
+    let baseURL: BaseURLProvider
```

Comment:
private

Reply:
Cannot be because I use it in the view

Reply:
Oh, now I see why. It's required by some further, nested views and because of that it needs to be exposed here. Sad. A comment would be nice explaining that